### PR TITLE
add hardhat-circom plugin

### PR DESCRIPTION
**Pull request by jacobrosenthal**
Happy to bikeshed anything. Thanks

### CODE REVIEW

File: docs/.vuepress/plugins.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const plugins = [
+  {
+    name: "hardhat-circom",
+    author: "Project Sophon",
+    authorUrl: "https://github.com/projectsophon",
+    url:
+      "https://github.com/projectsophon/hardhat-circom",
```
Comment (alcuadrado) [inline docs/.vuepress/plugins.js:7]:
```suggestion
      "https://github.com/projectsophon/hardhat-circom/tree/master",
```

This is annoying, but our script requires a link to the branch.

**Comment by alcuadrado** [comment]
Thanks for sending this PR! And congrats on launching the plugin.

I just left a small suggestion.

**Comment by alcuadrado** [comment]
Oh, also, can you point this to `website`?

**Comment by jacobrosenthal** [comment]
Thanks. Id love to take a call with your team sometime to see where you're headed and where the holes are. 

Ready for squash

**Comment by fvictorio** [comment]
@alcuadrado I think we have an (unwritten?) rule that the community plugins go after the official ones, right? At least until we do something like ordering them by the number of downloads or whatever.

**Comment by jacobrosenthal** [comment]
Moved to bottom.

**Comment by alcuadrado** [comment]
Great, thanks! I think I'll implement that sorting logic now.

**Pull request merged**